### PR TITLE
internal/vendoring quiver

### PR DESCRIPTION
**Pull request by tglstory**
@theevocator

### CODE REVIEW

**Comment by tglstory** [comment]
I did not understand what I wanted to do in the first commit. made changes that make more sense

File: cmd/listserving/list-collections.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 	"strings"
 
 	"github.com/foursquare/fsgo/net/thriftrpc"
-	"github.com/foursquare/quiver/gen"
+	"quiver/internal/gen"
```
Comment (theevocater) [inline cmd/listserving/list-collections.go]: I think you need to remove all the internal changes now that you reverted that bit of your original patch

**Comment by tglstory** [comment]
@theevocater Done. I wonder how that passed build... I guess go build doesn't build subprojects?

**Comment by theevocater** [comment]
no idea, maybe it didn't see them as 'changed'?

**Comment by theevocater** [comment]
sweet! could you rebase this into one commit?

**Pull request merged**